ega-download-wf/main.nf

```diff
@@ -124,6 +127,7 @@ workflow EgaDownloadWf {
 
       decryptAspera(
         downloadAspera.out.output_file,
+        name_ch,
         file(params.c4gh_secret_key)
```

Comment:
I don't think this will work properly. 
Assume that we have 2 ega files to be downloaded, 2 concurrent `downloadAspera` processes will be kicked off, and they output to two separate queue channels. 
Then `decryptAspera` should collect all `output_file` from these two separate queue channels, otherwise it won't work.

Reply:
I would suggest:
1. move `name_ch` to the inputs of `downloadAspera`, as this can guarantee the inputs to `downloadAspera` are paired with each other. 
2. rename the output file at the end of `downloadAspera`
3. `decryptAspera` should accept `downloadAspera.out.output_file.collect()` as one of the input, and loop through  all the input files to do decryption.

Reply:
@edsu7 , maybe another option is to `collect` all `downloadAspera` process, then `flatten` so that we can kick off separate `decryptAspera` process to decrypt each file. 
```
downloadAspera.out.output_file.collect().flatten().set{ new_ch }
decryptAspera( new_ch,  file(params.c4gh_secret_key)
```
That way, if any of the file decryption failed in between, it could still resume.